_api-reference/validate.md

```diff
@@ -0,0 +1,205 @@
+allow_partial_search_results | Boolean | Whether to return partial results if the request runs into an error or times out. Default is `true`.
+`analyzer` | String | The analyzer to use in the query string. This should only be used with the `q` option.
+`analyze_wildcard` | Boolean | Specifies whether to analyze wildcard and prefix queries. Default is `false`. 
+`default_operator` | String | Indicates whether the default operator for a string query should be `AND` or `OR`. Default is `OR`.
+`df` | String | The default field in case a field prefix is not provided in the query string.
+`expand_wildcards` | String | Specifies the type of index that wildcard expressions can match. Supports comma-separated values. Valid values are `all` (match any index), `open` (match open, non-hidden indexes), `closed` (match closed, non-hidden indexes), `hidden` (match hidden indexes), and `none` (deny wildcard expressions). Default is `open`.
+`explain` | Boolean | Whether to return details about how OpenSearch computed the [document's score](#explain). Default is `false`.
+`ignore_unavailable` |  Boolean | Specifies whether to include missing or closed indexes in the response and ignores unavailable shards during the search request. Default is `false`.
+`lenient` | Boolean | Specifies whether OpenSearch should ignore format-based query failures (for example, querying a text field for an integer). Default is `false`. 
+`rewrite` | Determines how OpenSearch [rewrites](#rewrite) and scores multi-term queries. Valid values are `constant_score`, `scoring_boolean`, `constant_score_boolean`, `top_terms_N`, `top_terms_boost_N`, and `top_terms_blended_freqs_N`. Default is `constant_score`.
+`q` | String | Query in the Lucene query string syntax.
+
+## Example request
+
+The following example uses an index named `Hamlet`, created using the following `bulk` request:
+
+```
+PUT hamlet/_bulk?refresh
+{"index":{"_id":1}}
+{"user" : { "id": "hamlet" }, "@timestamp" : "2099-11-15T14:12:12", "message" : "To Search or Not To Search"}
+{"index":{"_id":2}}
+{"user" : { "id": "hamlet" }, "@timestamp" : "2099-11-15T14:12:13", "message" : "My dad says that I'm such a ham."}
+```
+{% include copy.html %}
+
+You can then use the Validate Query API to validate a query made against the index, as shown in the following example:
+
+```
+GET hamlet/_validate/query?q=user.id:hamlet
+```
+{% include copy.html %}
+
+The query can also be sent as a request body, as shown in the following example:
+
+```
+GET hamlet/_validate/query
+{
+  "query" : {
+    "bool" : {
+      "must" : {
+        "query_string" : {
+          "query" : "*:*"
+        }
+      },
+      "filter" : {
+        "term" : { "user.id" : "hamlet" }
+      }
+    }
+  }
+}
+```
+{% include copy.html %}
+
+
+## Example responses
+
+Depending on the validity of the request query, OpenSearch responds that the query is true, as shown in the following example response where the `valid` parameter is shown as `true`:
```

Comment:
Can we clarify/simplify here a bit? The second phrase isn't necessarily true in the context of the first. Do we mean something like "If the query passes validation, then the response indicates that the query is `true`, as shown in the following example response, where the `valid` parameter is `true`:"?